CONTRIBUTING.md

```
@@ -193,6 +193,34 @@ API integration tests against MariaDB use `docker compose -f docker-compose.test
+4. Run migrations and conformance tests locally:
+
+```bash
+docker compose -f docker-compose.test.yml up -d --wait
+export TANJUN_TEST_DB_HOST=127.0.0.1 TANJUN_TEST_DB_PORT=3307
+export TANJUN_TEST_DB_USER=test_user TANJUN_TEST_DB_PASSWORD=test_password
+export TANJUN_TEST_DB_NAME=tanjun_test
+alembic upgrade head
+pytest tests/integration/api/test_schema_conformance.py -v
+```
+
+5. Update [`docs/database-schema.md`](docs/database-schema.md) when columns or tables change.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)